packages/@internationalized/date/docs/ZonedDateTime.mdx

```diff
@@ -0,0 +1,486 @@
+
+## Introduction
+
+A `CalendarDateTime` object represents an exact date and time in a specific time zone, in a specific calendar system such as the Gregorian calendar. Use this type to represent an exact moment in time at a particular location on Earth.
+
+A `CalendarDateTime` can be created using the constructor. This example creates a date that represents February 3rd, 2022 at 9:15 AM in the Gregorian calendar system.
+
+```tsx
+import {CalendarDateTime} from '@internationalized/date';
+
+let date = new ZonedDateTime(
+  // Date
+  2022, 2, 3,
+  // Time zone and UTC offsset
+  'America/Los_Angeles', -28800000,
+  // Time
+  12, 24, 45
+);
+```
+
+You can also create a `CalendarDate` by parsing an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) formatted string using the one of the following functions:
+
+* <TypeLink links={docs.links} type={docs.exports.parseZonedDateTime} /> – This function parses a date with an explicit time zone and optional UTC offset attached (e.g. `"2021-11-07T00:45[America/Los_Angeles]"` or `"2021-11-07T00:45-07:00[America/Los_Angeles]"`). This format preserves the maximum amount of information. If the exact local time and time zone that a user selected is important, use this format. Storing the time zone and offset that was selected rather than converting to UTC ensures that the local time is correct regardless of daylight saving rule changes (e.g. if a locale abolishes DST). Examples where this applies include calendar events, reminders, and other times that occur in a particular location.
+* <TypeLink links={docs.links} type={docs.exports.parseAbsolute} /> – This function parses an absolute date and time that occurs at the same instant at all locations on Earth. It can be represented in UTC (e.g. `"2021-11-07T07:45:00Z"`), or stored with a particular offset (e.g. `"2021-11-07T07:45:00-07:00"`). A time zone identifier, e.g. `America/Los_Angeles`, must be passed, and the result will be converted into that time zone. Absolute times are the best way to represent events that occurred in the past, or future events where an exact time regardless of time zone is needed.
+* <TypeLink links={docs.links} type={docs.exports.parseAbsoluteToLocal} /> – This function parses an absolute date and time into the current user's local time zone. It is a shortcut for `parseAbsolute`, and accepts the same formats.
+
+```tsx
+import {parseZonedDateTime} from '@internationalized/date';
+
+let date = parseZonedDateTime('2022-11-07T00:45[America/Los_Angeles]');
+let date = parseAbsolute('2021-11-07T07:45:00Z', 'America/Los_Angeles');
+let date = parseAbsoluteToLocal('2021-11-07T07:45:00Z');
+```
+
+Once you have a `ZonedDateTime` object, you can read its properties, or manipulate it as described in the [Manipulating dates](#manipulating-dates) section below. You can also convert it to an ISO 8601 string, native JavaScript `Date` object, or another representation. See the [Conversion](#conversion) section below for details.
```

Comment:
Should this `ISO 8601` be a link?

Reply:
Seems unnecessary to repeat since there is already a link on the first instance above?